apps/amo/models.py

```diff
@@ -368,6 +368,11 @@ def reload(self):
                 pass
         return self
 
+    @classmethod
+    def get_unfiltered_manager(cls):
```

Comment:
Actually it's used in three different places: twice in this file, and once in `apps/devhub/models.py`